src/uniqBy.js

```diff
@@ -36,12 +36,12 @@ module.exports = _curry2(/* globals Set */ typeof Set === 'undefined' ?
   } :
   function uniqBySet(fn, list) {
     var set           = new Set(),
-        applied       = [], appliedItem, item,
-        prevSetSize   = 0,  newSetSize,
-        result        = [],
-        nullExists    = false,
-        negZeroExists = false,
-        idx           = 0;
+      applied       = [], appliedItem, item,
+      prevSetSize   = 0,  newSetSize,
+      result        = [],
+      nullExists    = false,
+      negZeroExists = false,
+      idx           = 0;
```

Comment:
Sorry, I meant the `=` before `new Set()` is no longer vertically aligned with the other `=`.